packages/algorand-js/src/types.ts

```diff
@@ -125,3 +125,7 @@ export enum EncodingType {
 export interface AccountsMap {
   [addr: string]: AccountInfo
 }
+
+export interface ApplicationMap {
+  [appId: number]: SSCParams
+}
```

Comment:
Why not using a `Map` type?
eg:
```
Map<number, SSCParams>  // instead of ApplicationMap
Map<string, AccountInfo>  // instead of AccountsMap
```

Reply:
Actually we are fetching info from nested objects in global and local state. So using `Object` key - value pairs seemed easier.
eg.
```
this.interpreter.globalApps[appId]["global-state"]
```
Should i change to `Map`?

Reply:
Update: Changed to using `Map`